step-grid-parent/step-grid-server/src/main/java/step/grid/GridImpl.java

```diff
@@ -159,6 +166,40 @@ public Map<String, String> getTokenAffinityEvaluatorProperties() {
 		public void setTokenAffinityEvaluatorProperties(Map<String, String> tokenAffinityEvaluatorProperties) {
 			this.tokenAffinityEvaluatorProperties = tokenAffinityEvaluatorProperties;
 		}
+
+		public boolean isDeferAcceptingRegistrationMessages() {
+			return deferAcceptingRegistrationMessages;
+		}
+
+		public void setDeferAcceptingRegistrationMessages(boolean deferAcceptingRegistrationMessages) {
+			this.deferAcceptingRegistrationMessages = deferAcceptingRegistrationMessages;
+		}
+	}
+
+	public interface GridImplConfigModifier {
+		void modifyConfig(GridImplConfig config);
+	}
+
+	public static class GridImplConfigModifiers extends LinkedList<GridImplConfigModifier> {}

```

Comment:
@cl-exense What is the purpose of this class?

Reply:
You're right, it does not belong here. moved to step instead.